### PR TITLE
update to latest lts build.  16.17.0

### PR DESCRIPTION
**Pull request by KevinLaMS**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/75818)

### CODE REVIEW

File: manifests/o/OpenJS/NodeJS/16.17.0/OpenJS.NodeJS.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+PackageVersion: 16.17.0
+PackageLocale: en-US
+Publisher: Node.js Foundation
+PublisherUrl: https://nodejs.org
+PublisherSupportUrl: https://github.com/nodejs/help/issues
+PrivacyUrl: https://privacy-policy.openjsf.org/
+Author: OpenJS Foundation
+PackageName: Node.js
+PackageUrl: https://nodejs.org/en/
+License: Copyright (c) Joyent, Inc. and other Node contributors. All rights reserved.
```
Comment (Trenly) [inline manifests/o/OpenJS/NodeJS/16.17.0/OpenJS.NodeJS.locale.en-US.yaml:14]: A copyright isn't a license

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Looks like Node.js LTS version 16.17.0 exists under https://github.com/microsoft/winget-pkgs/blob/master/manifests/o/OpenJS/NodeJS/LTS/16.17.0/OpenJS.NodeJS.LTS.installer.yaml#L15-L23

**Comment by ItzLevvie** [comment]
Close with reason: Node.js LTS version 16.17.0 is available under OpenJS.NodeJS.LTS;

**Comment by jsoref** [comment]
Hi @KevinLaMS, @Trenly: you're going to want to delete this branch

**Pull request closed without merging**